examples/http-request-latency-burnrate.jsonnet

```diff
@@ -0,0 +1,18 @@
+local latency = import '../slo-libsonnet/latency-burn.libsonnet';
+
+{
+  local query = latency.burn({
+    metric: 'http_request_duration_seconds_bucket',
+    selectors: ['job="telemeter-server"','handler="upload"'],
+    # How much responce delay is too much.
+    latencyTarget: "1",
+    # The 30 days SLO promise.
+    # When the promise is 99% that means that
+    # in 30d can only have 1% quries above the latencyTarget.
+    latencyBudget: 100-99,
```

Comment:
This should be consistent with the error-burnrates. In that one we give the budget as 1-0.99, so percentage between 0 - 1 rather than 0-100. 
Then we can get rid of a few instances where you currently multiply by 100.

Reply:
Or would you rather make the change in the other component to make it got from 0 - 100?

Reply:
> This should be consistent with the error-burnrates. In that one we give the budget as 1-0.99, so percentage between 0 - 1 rather than 0-100.

no you can still have values from 0 to 100. In this case the budget is 1% because the SLO is 99% but if you have an SLO 99.9 you would than 100-99.9 and have a budget of 0.1%

Reply:
The only question is, does our budget get defined as 0-1 or 0-100? Right now on the error burnrate, we use 0-1.

Reply:
hm , I think it is more logical to define it from 0 to 100

Reply:
I'd say, let's go with 0-1 for consistency with the rest for now and have another discussion about changing that afterwards.